### ISSUE TITLE
Prove new stage- and destination concept with UI-test

### ISSUE THREAD

**Issue by matthiasrohmer** [jungvonmatt/amp.dev#84]
## Objective

The new stage with the rolling destinations should be UI tested before implementing them.

## Background

TBD.

## Tasks
- [x] Setup refined principal dummy
- [x] Define content that should be explored by users
- [x] Perform user test

**Comment by LongoOfficial** [comment]
As requested, we tested the GUI within the first view port of stage (home and about AMP), if the user would understand whether elements are clickable or not and would understand the UX pattern behind it.

In general all of our test users were able to tell apart when the rolling destination UI element was clickable (about AMP stage) and not clickable (home stage). In an additinal dialogue we found out, that the arrow element next to the destination word was strong enough to indicate the drop down.

With just testing the stage of home and about amp, some users were confused with the destination filtering, but also some others could surely tell what would happen, when selecting a specific destination.

For the home stage one test user was expecting, when clicking on "get stated" button, to get to the specific topic the rolling destination element was displaying in that particular time frame.

![stage_home](https://user-images.githubusercontent.com/41894292/45409255-fce00e00-b66e-11e8-92df-19d0708341f5.jpg)
![stage_aboutamp](https://user-images.githubusercontent.com/41894292/45409256-fce00e00-b66e-11e8-8e2f-f54aefd42762.jpg)

[stageUI_test.zip](https://github.com/jungvonmatt/amp.dev/files/2374000/stageUI_test.zip)
